src/Audacia.ExceptionHandling.AspNetCore/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ﻿# Changelog
 
+## 5.1.2 - 2024-12-10
+### Added
+- No new functionality added
+
+### Removed
+- Individual project versions
```

Comment:
I don't think you need to say this in the removed, the bit in Changed covers it

Reply:
Good point, I've removed that